Core/MIPS/RiscV/RiscVCompFPU.cpp

```diff
@@ -51,7 +51,35 @@ void RiscVJit::CompIR_FArith(IRInst inst) {
 	case IROp::FMul:
 		fpr.MapDirtyInIn(inst.dest, inst.src1, inst.src2);
 		if (inst.src1 != inst.src2) {
-			CompIR_Generic(inst);
+			// These will output 0x80/0x01 if infinity, 0x10/0x80 if zero.
```

Comment:
Hm, on ARM64 we just emit an FMUL instruction, since it's behavior in inf*0 is the same as for MIPS, [unlike x86](https://github.com/unknownbrackets/ppsspp/commit/5afc020fbdbf6e9f9139fbc4919430af3a652285). I guess you noticed through tests that RISC-V is closer to x86 here?

Reply:
Well, I was worried because it doesn't specify much and there's different people implementing RISC-V.  Maybe we can remove, but I'd want to have more confidence in it working on more chips first... I didn't actually verify if FMUL was doing this or not.

-[Unknown]

Reply:
Okay, that's reasonable. Please add that note as a code comment though :)